_sections/30-projects.md

```diff
@@ -27,5 +27,6 @@ style: center
 - [Altair](https://github.com/ellisonbg/altair)
 - [gala](https://gala.readthedocs.io)
 - [cual-id](https://github.com/johnchase/cual-id)
+- [music21](https://web.mit.edu/music21/)
```

Comment:
I get a 'certificate error' from the https site, but changing it to http works.

Also, we list projects in rough order of prominence, using Github stars as a metric where possible. Looking at [this repo](https://github.com/cuthbertlab/music21), I think you can go just under readiab.